wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -255,7 +255,7 @@ def visit_Assign(self, node: ast.Assign) -> None:
 
         """
         self._check_assign_targets(node)
-        if isinstance(node.targets[0], ast.Tuple):
+        if isinstance(node.targets[0], (ast.Tuple, ast.List)):
```

Comment:
Can you please also update tests for other violations that are touched by this change?